yb-voyager/cmd/exportData.go

```diff
@@ -181,7 +176,7 @@ func exportDataOnline() bool {
 	return false
 }
 
-func validateTableListFlag(tableListString string) {
+func validateTableListFlag(tableListString string, excludeList bool) {
```

Comment:
How about taking `flagName` as argument instead of `excludeList bool`?
That can simplify the error reporting.